webapp/app/[locale]/tunnel/_components/evmDeposit.tsx

```diff
@@ -6,15 +6,18 @@ import { CustomTunnelsThroughPartner } from 'components/customTunnelsThroughPart
 import { EvmFeesSummary } from 'components/evmFeesSummary'
 import { useNativeTokenBalance, useTokenBalance } from 'hooks/useBalance'
 import { useChain } from 'hooks/useChain'
+import { useHemi } from 'hooks/useHemi'
 import { useNetworkType } from 'hooks/useNetworkType'
 import dynamic from 'next/dynamic'
 import { useTranslations } from 'next-intl'
 import { useEffect, useState } from 'react'
+import { getTunnelContracts } from 'utils/crossChainMessenger'
 import { getNativeToken, isNativeToken } from 'utils/nativeToken'
 import { tunnelsThroughPartner } from 'utils/token'
 import { walletIsConnected } from 'utils/wallet'
 import { formatUnits } from 'viem'
 import { useAccount as useEvmAccount } from 'wagmi'
+import { useAllowance } from 'wagmi-erc20-hooks/useAllowance'
```

Comment:
```suggestion
import { useAllowance } from 'wagmi-erc20-hooks'
```

This ☝🏽  seems to be causing the build to fail

Reply:
Thank you! Fixed it